ecosystem/sep-0024.md

```diff
@@ -7,7 +7,7 @@ Author: SDF
 Status: Active
 Created: 2019-09-18
 Updated: 2021-01-19
-Version 1.2.2
+Version 1.3.0
```

Comment:
❓ I'm not sure what version this should be. It's not a patch because it's not simply fixing a bug. It's not really new functionality that stands alone, it is a change to existing functionality that is sort of breaking.

What happens if an anchor does not fulfill its responsibilities to close the window or provide a way for the user to close it? Can client integrations become stuck?

Reply:
I believe it's non-breaking because the user can always exit the interactive flow, both browser popups and mobile webviews have close buttons regardless of the content served in them.